assets/multiqc_config.yml

```
@@ -39,6 +39,10 @@ sp:
   diamond:
     contents: "diamond v"
     num_lines: 10
+  fastqc/data:
+    fn_re: ".*[fastqc|falco]_data.txt"
```

If this gets translated to a Python regex, the square brackets actually mean a group of those characters and not those exact strings. Maybe the following would work:

```suggestion
    fn_re: "(fastqc|falco)_data.txt$"
```

---

Alternatively, maybe a simpler glob but I'm not sure this will work.

```suggestion
    fn: "*{fastqc,falco}_data.txt"
```

---

The glob didn't work for picking up Falco unfortunately, but the improved python one did :)